### PR TITLE
WAS-337 Limited Company name page

### PR DESCRIPTION
**Pull request by woodpigeon**
Implement the journey for a Limited Company user
* capture name and company number (crn)
* display limited company name and number on check details page
* display limited company name and number on confirmation email

### CODE REVIEW

File: spec/factories/enrollment_factory.rb [hunk 1]
```diff
@@ -127,4 +126,22 @@
       after(:create) { |object| object.register! }
     end
   end
+
+  # These factories were an attempt to get complete enrollments without having to understand
```
Comment (Cruikshanks) [inline spec/factories/enrollment_factory.rb]: Does this need a ```TODO:``` added so we can pick it up later?

File: spec/models/prepare_data_for_review_spec.rb [hunk 2]
```diff
@@ -6,11 +6,11 @@
       let(:enrollment)        { create(:page_reviewing) }
       let(:enrollment_review) { DigitalServicesCore::EnrollmentReview.new(enrollment) }
 
-      it "can find all states with a section heading in locales" do
-        headings = enrollment_review.find_section_headings
-        expect(headings).to be_instance_of Array
-        expect(headings.size).to be > 1
-      end
+      # it "can find all states with a section heading in locales" do
```
Comment (Cruikshanks) [inline spec/models/prepare_data_for_review_spec.rb]: Is this a candidate for deletion?

**Comment by Cruikshanks** [comment]
Just a couple of comments and then I'll tackle the squash @woodpigeon

**Comment by woodpigeon** [comment]
@Cruikshanks removed unused code as per comments - over to you again

**Pull request merged**